src/lib/static/css/stateRecidivismTable.css

```diff
@@ -1,2 +1 @@
+.columns .ui-table{border-collapse:collapse;width:100%;margin:1em 0;background-color:transparent;border:solid 1px #D5D5D5}.columns .ui-table thead{background-color:#E5E5E5}.columns .ui-table thead th{background-color:transparent;color:#222222;padding:10px;border-right:solid 1px #D5D5D5;cursor:pointer}.columns .ui-table thead th.ui-table-sort-up,.columns .ui-table thead th.ui-table-sort-down{background-color:#CCCCCC}.columns .ui-table thead th:last-child{border-right:0px}.columns .ui-table thead th .ui-arrow{float:right;font-size:10px}.columns .ui-table tbody{background-color:transparent}.columns .ui-table tbody tr.ui-table-rows-odd{background-color:#F2F2F2}.columns .ui-table tbody tr.ui-table-rows-even{background:#FFFFFF}.columns .ui-table tbody tr td{background:transparent;border-right:solid 1px #D5D5D5;color:#656565;padding:10px}.columns .ui-table-controls span{font-size:12px;padding:5px;vertical-align:middle}.columns .ui-table-controls span.ui-table-control-next,.columns .ui-table-controls span.ui-table-control-prev{cursor:pointer;font-family:"Arial Narrow";font-size:16px}.columns .ui-table-controls span.ui-table-control-disabled{color:#999999;font-family:"Arial Narrow";font-size:16px}.columns .ui-table-footer{width:100%;padding:8px 0;font-size:11px;text-align:left;color:#333}.columns .ui-table-footer span{vertical-align:middle}.columns .ui-table-footer .ui-table-size,.columns .ui-table-footer .ui-table-results,.columns .ui-table-footer .ui-table-controls{box-sizing: border-box;display:inline-block;width:32%}.columns .ui-table-footer .ui-table-size{padding-left:20px}.columns .ui-table-footer .ui-table-results{text-align:center}.columns .ui-table-footer .ui-table-controls{text-align:right}.columns .ui-table-footer .ui-table-control-next,.columns .ui-table-footer .ui-table-control-prev,.columns .ui-table-footer .ui-table-control-disabled{display:inline-block;background-color:transparent;padding:5px;vertical-align:middle;cursor:pointer;text-align:center}.columns .ui-table-footer .ui-table-control-disabled img{opacity:0.5}.columns .ui-columns-search{text-align:right}.columns .ui-columns-search input{width:200px;border-radius:10px;padding:4px 10px 4px 25px;border:2px solid #ccc;background-image:url(../../../../static/images/search.png);background-position:5px center;background-repeat:no-repeat}.columns .ui-columns-search input:focus{border:2px solid #6196CD;outline:none}
```

Comment:
There is a slight CSS change in this line. `box-sizing: border-box` was added to `.ui-table-controls`. Is that desired?

Reply:
Yes, this was intentional. This was necessary to fix the issue with the page controls of RecidivizTable rendering on a second line on a smaller viewport. In the original code, this style was being applied to the entire HTML document which I don't think we want to do as it could have unintended side effects. So, I included it at the most specific level possible.